src/shared/components/community/sidebar.tsx

```diff
@@ -144,10 +145,10 @@ export class Sidebar extends Component<SidebarProps, SidebarState> {
             {myUSerInfo && this.blockCommunity()}
             {!myUSerInfo && (
               <div className="alert alert-info" role="alert">
-                {i18n.t("community_not_logged_in_alert", {
-                  community: name,
-                  instance: hostname(actor_id),
-                })}
+                <T i18nKey="community_not_logged_in_alert">#</T>
+                <code className="user-select-all">
+                  !{name}@{hostname(actor_id)}
+                </code>
```

Comment:
```suggestion
<T i18nKey="community_not_logged_in_alert" interpolation={{community: name, instance: hostname(actor_id)}}>
    #<code>#</code>
</T>
```

You'll need to change your translation in the translations repo to be like this:
`"You are not logged in. However you can subscribe from another Fediverse account, for example Lemmy or Mastodon. To do this, paste the following into the search field of your instance: <1>!{{community}}@{{instance}}</1>"`

Reply:
I see, thanks.